src/functions/template-tags/general.php

```diff
@@ -9,11 +9,15 @@
 use Tribe__Timezones as Timezones;
 
 /**
- * Includes a view file, runs hooks around the view
+ * Retrieves and includes the template for a specified view.
  *
- * @param bool|string $view View slug
+ * This function will trigger deprecation warnings if used with certain views, or without specifying a view.
+ * It also includes hooks for actions before and after the view template is loaded.
  *
- **/
+ * @param string|bool $view Optional. The view to retrieve the template for. Default false.
```

Comment:
Might be worth adding that leaving it to default - not passing a view, will throw a deprecation warning.

Reply:
Unless we're planning on actually removing the deprecation notices (it's past 6.0.0)

Reply:
Good point @Camwyn 

I just updated the message